glassy_main_node:
  ros__parameters:
    publishers:
      state: true
      battery: false
    subscriptions:
      offboard_direct: true
      offboard_attitude_rate: false
      manual_control: false
    data_rates:
      position: 20
      odometry: 50
      attitude: 50 

    
    

